test/test_grocy.py

```diff
@@ -308,15 +354,21 @@ def test_add_product_error(self):
 
     @responses.activate
     def test_consume_product_valid(self):
-        data = {
-            "amount": 1.3,
-            "spoiled": False,
-            "transaction_type": str(TransactionType.CONSUME)
-        }
-        responses.add(responses.POST, f"{CONST_BASE_URL}:{CONST_PORT}/api/stock/products/1/consume", json=data, status=200)
+        responses.add(responses.POST, f"{CONST_BASE_URL}:{CONST_PORT}/api/stock/products/1/consume", status=200)
         self.assertIsNone(self.grocy.consume_product(1, 1.3, datetime.now()))
 
     @responses.activate
     def test_consume_product_error(self):
         responses.add(responses.POST, f"{CONST_BASE_URL}:{CONST_PORT}/api/stock/products/1/consume", status=400)
         self.assertRaises(HTTPError, self.grocy.consume_product, 1, 1.3, datetime.now())
+
+    @responses.activate
+    def test_execute_chore_valid(self):
+        date_exec_chore = datetime.now()
```

Comment:
This shouldn't be needed that way as the parameter is `datetime.now()` by default. Either change this to a fixed value (preferred) or remove this. Also valid for the remaining `datetime.now()` in the tests.

Reply:
For coverage you need to pass all parameters even if they have a default value.
I will correct with a fixed value